spec/support/active_record.rb

```diff
@@ -10,9 +10,16 @@
 config = Travis.config.database.to_h
 config.merge!('adapter' => 'jdbcpostgresql', 'username' => ENV['USER']) if RUBY_PLATFORM == 'java'
 
+logs_config = config.clone.merge(:database => "travis_logs_#{Travis.env}")
```

Comment:
Should we use Travis.config.logs_database.to_h here like on line 10 and ensure there is a logs_database config added to Travis.config?